### PR TITLE
{numlib}[intel/2019b] arpack-ng v3.7.0

### PR DESCRIPTION
**Pull request by Falkor**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Falkor** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-067 - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.7.4
See https://gist.github.com/85ce3dd87740dfef349f7e5774289c17 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node150.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/a76aa69197f535142222439fe5e7b85f for a full test report.

File: easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.7.0-add_include_CheckSymbolExists_CMakeLists_txt.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+diff -Nru arpack-ng-3.7.0.orig/CMakeLists.txt arpack-ng-3.7.0/CMakeLists.txt
```
Comment (lexming) [inline easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.7.0-add_include_CheckSymbolExists_CMakeLists_txt.patch:1]: Please add a description of this patch and your authorship.

File: easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.7.0-intel-2019b.eb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    ('CMake', '3.15.3')
+]
+
+dependencies = [
+    ('Eigen', '3.3.7', '', True)
+]
+
+separate_build_dir = True
+
+local_common_configopts = "-DICB=ON -DMPI=ON -DEXAMPLES=ON"
```
Comment (lexming) [inline easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.7.0-intel-2019b.eb:32]: I suggest to change the installation directory of the libraries to match the installations of other `arpack-ng` easyconfigs
```suggestion
local_common_configopts = "-DCMAKE_INSTALL_LIBDIR=lib -DICB=ON -DMPI=ON -DEXAMPLES=ON"
```

File: easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.7.0-intel-2019b.eb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+# perform iterative build to get both static and shared libraries
+configopts = [
+    local_common_configopts + ' -DBUILD_SHARED_LIBS=OFF',
+    local_common_configopts + ' -DBUILD_SHARED_LIBS=ON',
+]
+
+sanity_check_paths = {
+    'files': ["lib64/libarpack.a",  "lib64/libarpack.%s" % SHLIB_EXT,
+              "lib64/libparpack.a", "lib64/libparpack.%s" % SHLIB_EXT],
```
Comment (lexming) [inline easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.7.0-intel-2019b.eb:42]: Hence the sanity checks would have to be changed from `lib64` to `lib`

**Comment by Falkor** [comment]
Test report by @ULHPC
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
iris-003 - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.7.4
See https://gist.github.com/c59ebc58973afbcf12dfca3a304076bf for a full test report.

**Comment by Falkor** [comment]
Test report by @ULHPC
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
iris-003 - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.7.4
See https://gist.github.com/e55ed8e4bd3f8bdbeb1c58e2de5802b7 for a full test report.

**Comment by Falkor** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-003 - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.7.4
See https://gist.github.com/65fb759bb88bd63be3f635005c62858c for a full test report.

File: easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.7.0-intel-2019b.eb (hunk 3 above)
Comment (Falkor) [reply easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.7.0-intel-2019b.eb]: should be fine now. 
Sorry for the mess in the commits, I should have been more careful...

File: easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.7.0-add_include_CheckSymbolExists_CMakeLists_txt.patch (hunk 1 above)
Comment (Falkor) [reply easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.7.0-add_include_CheckSymbolExists_CMakeLists_txt.patch:3]: done in 3a7e150

**Comment by lexming** [comment]
Re-triggering travis...

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node372.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/e901b81724761abab5818b17a88c7717 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @Falkor !

**Pull request merged**